rnd/autogpt_server/autogpt_server/data/model.py

```diff
@@ -109,6 +109,7 @@ def SchemaField(
     title: Optional[str] = None,
     description: Optional[str] = None,
     placeholder: Optional[str] = None,
+    non_advanced: Optional[bool] = None,
```

Comment:
Using double negatives on bools is confusing. It would be better to refer to this as simple/basic/core if you want the bool this way around or if you are okay with inverting the checks, just using advanced